web_timeline/readme/CONTRIBUTORS.rst

```diff
@@ -13,3 +13,6 @@
 * `Onestein <https://www.onestein.nl>`_:
     * Dennis Sluijk <d.sluijk@onestein.nl>
     * Anjeel Haria
+* `XCG Consulting <https://xcg-consulting.fr>`_:
```

Comment:
Incorrect RST format:

```suggestion

* `XCG Consulting <https://xcg-consulting.fr>`_:
```

Reply:
[added](https://github.com/OCA/web/compare/0b200aae387ad4579fce253b6e54525ed56f5c10..a47659fda80aaa3e8f5137c5abe6c7b3105ba998)

I wasn't sure because there are 2 other companies in that list where the empty line is missing - we should add them too I guess

Reply:
Tecnativa has the empty line. About Onestein, it's working somehow due to the extra 2 spaces in the following lines, but you can see in the GitHub rendering that it's shown weirdly. You can homogenize everything as it should be.

Reply:
[done](https://github.com/OCA/web/compare/a82d7fb8b4317876867ff8fd91c1b422fa55766a..bc577b6656cfe747251b522d96aa027bb73dd4c7), should look better now